gluten-flink/planner/src/main/java/org/apache/gluten/rexnode/functions/ModRexCallConverter.java

```diff
@@ -27,6 +27,7 @@
 
 import org.apache.calcite.rex.RexCall;
 
+import java.util.Arrays;
 import java.util.List;
 
 public class ModRexCallConverter extends BaseRexCallConverter {
```

Comment:
Seems this class can extend `BasicArithmeticOperatorRexCallConverter` to reuse its `isSuitable`. If `toTypedExpr` method is really different in getting result type, just override this method in `ModRexCallConverter`.